src/main.js

```diff
@@ -80,11 +78,35 @@ function removeButtonState() {
   saveButton.innerHTML = "Save";
 }
 
-function deleteIdea(event) {
+function checkTarget(event) {
   event.preventDefault()
   if (event.target.classList.contains("delete-idea")) {
-    updateIdeaArray(event.target.parentElement.parentElement.id);
```

Comment:
Refactor for renaming this pathway `parentElement.parentElement` throughout project per developer empathy.